dispatch.go

```diff
@@ -273,6 +281,17 @@ func (ag *aggrGroup) String() string {
 	return fmt.Sprintf("%v", ag.fingerprint())
 }
 
+func (ag *aggrGroup) alertSlice() []*types.Alert {
+	ag.mtx.RLock()
+	defer ag.mtx.RUnlock()
+
+	var alerts []*types.Alert
```

Comment:
Actually, the most idiomatic way is probably the following (in lieu of lines 288–292):

`return append(make([]*types.Alert, 0, len(ag.alerts)), ag.alerts...)`